frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -129,7 +136,7 @@ const AdvancedAutoFilter: FC = () => {
               label="Regions"
               onChange={handleRegionChange}
               value={getValueById(options.regions, filters.regionId)}
-              options={options?.regions?.map((item: AutoRiaOption) => ({
+              options={options.regions.map((item: AutoRiaOption) => ({
```

Comment:
Is the removal of the optional chaining expected? Do we know that the `options` and `regions` will be defined? Why the `?.` was added in the first place?

Reply:
I already use main check of existing options in line 126 `if (isLoading) return <h1>Loading...</h1>;`
Also I use conditional rendering in every mounting of those components.  Probably I added `?.`  before just to quickly fix TSerrors.